aruco_detect/src/aruco_detect.cpp

```diff
@@ -238,18 +238,23 @@ void FiducialsNode::camInfoCallback(const sensor_msgs::CameraInfo::ConstPtr& msg
         return;
     }
 
-    for (int i=0; i<3; i++) {
-        for (int j=0; j<3; j++) {
-            cameraMatrix.at<double>(i, j) = msg->K[i*3+j];
+    if (msg->distortion_model != "") {
+        for (int i=0; i<3; i++) {
```

Comment:
This doesn't look right: there should be a msg->K independently of whether there is a distortion model

Reply:
Thanks for doing the PR

The nodes should work with a K matrix but no distortion model, but should exit with an error if K is not provided.

Reply:
Maybe I am misunderstanding how the calibration works, but doesn't the K matrix not make sense if  there is a distortion model? Or is that only the D matrix?